chess-AI/experimental-work/NN Layout.py

```diff
@@ -0,0 +1,83 @@
+
+
+# this is the basic AI CNN layout
+# we will use this and improve it until it can be used for the final product.
```

Comment:
This should be a [module docstring](https://google.github.io/styleguide/pyguide.html#382-modules), and the docstring should be the first line in the file (no newlines at the top)